include/std/experimental/conqueue

```diff
@@ -421,7 +427,26 @@ template <typename T> struct buffer_queue<T>::pop_sender {
 
     friend void tag_invoke(stdexec::start_t, operation& op) noexcept {
       auto& self = op.queue;
+      auto&& env = stdexec::get_env(op.receiver);
+      auto stop_token = stdexec::get_stop_token(env);
+      constexpr bool sp = std::unstoppable_token<decltype(stop_token)>;
+
       std::unique_lock lock(self.mutex);
+      if constexpr (sp) {
+        if (stop_token.stop_requested()) {
+          lock.unlock();
+          stdexec::set_stopped(std::move(op.receiver));
+          return;
+        }
+      }
+
+      // Store and cache result here in case it changes during execution
+      const bool stop_possible = stop_token.stop_possible();
+      if constexpr (sp) {
```

Comment:
Here you are entering the if-branch only if the token is unstoppable.
I think you probably want to invert the initialisation of the `sp` variable above to be:
```
constexpr bool sp = !std::unstoppable_token<decltype(stop_token)>;
```

Reply:
Thank you. Addressed in https://github.com/GorNishanov/conqueue/pull/3